qa/L0_http/http_test.py

```diff
@@ -205,6 +207,31 @@ def test_content_encoding_unsupported_client(self):
                     ):
                         client.infer(model_name=model, inputs=inputs, headers=headers)
 
+    def test_descriptive_status_code(self):
+        model = "onnx_zero_1_float32_queue"
+        input_bytes = np.arange(8, dtype=np.float32).tobytes()
+
+        # Send two requests to model that only queues 1 request at the maximum,
+        # Expect the second request will be rejected with HTTP status code that
+        # aligns with error detail (server unavailable).
+        t = threading.Thread(
+            target=self._raw_binary_helper, args=(model, input_bytes, input_bytes)
+        )
+        t.start()
+        time.sleep(0.5)
+        try:
+            self._raw_binary_helper(model, input_bytes, input_bytes)
+        except requests.exceptions.HTTPError as ex:
+            self.assertEqual(
```

Comment:
1. Should the test be changed to expect exception to occur everytime? Looks like current test would pass if no exception occurs.
2. If so, will 0.5secs be flaky?

Reply:
> Should the test be changed to expect exception to occur everytime? Looks like current test would pass if no exception occurs.

Good catch, will fix to fail if there is no exception